tests/typecheck/fields/test_related.yml

```diff
@@ -692,6 +694,8 @@
         myapp/models:9: error: `.from_queryset` called from inside model class body
         myapp/models:11: note: Revealed type is "django.db.models.manager.RelatedManager[myapp.models.TransactionLog]"
         myapp/models:13: note: Revealed type is "Any"
+        myapp/models:14: note: Revealed type is "Any"
+        myapp/models:15: note: Revealed type is "Any"
```

Comment:
I think perhaps the test name should change? `foreign_key_relationship_for_models_with_custom_manager_unsolvable` -> `types_manager_as_any_when_manager_cant_be_resolved` (feel free to come up with another name). The test feels slightly outdated? I can't figure out what the expected outcome of the test is.

I also think the `Note, that we cannot resolve dynamic calls for custom managers:` should be removed..

Reply:
Augh, I was misinterpreting the initial meaning of the test. It should be split into two tests, see now this test (renamed to `reverse_for_manager_defined_in_class_body_falls_back_to_default_relatedmanager`) + new test in `test_managers.yml` (`manager_defined_in_class_body_replaced_with_any`)